sixtyfps_runtime/corelib/model.rs

```diff
@@ -205,14 +205,51 @@ impl Model for bool {
     }
 }
 
+/// A shared model
+#[repr(transparent)]
+#[derive(derive_more::Deref, derive_more::DerefMut)]
+pub struct SharedModel<T>(pub Rc<dyn Model<Data = T>>);
+
+impl<T> std::fmt::Debug for SharedModel<T> {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "SharedModel(dyn Model)")
+    }
+}
+
+impl<T> Clone for SharedModel<T> {
```

Comment:
I suppose this can also be achieved using `#[derive(Clone)]`?

Reply:
Oh, I see, same in ModelHandle, so not relevant for this change set :)

Reply:
the problem with #[derive(Clone)] is that it adds a `where T: Clone` which we don't want.